arch/risc-v/src/common/crt0.c

```diff
@@ -124,9 +189,11 @@ void _start(int argc, char *argv[])
 
   /* Call C++ constructors */
 
+  exec_ctors();
+
   /* Setup so that C++ destructors called on task exit */
 
-  /* REVISIT: Missing logic */
+  atexit(exec_dtors);
```

Comment:
This depends on: https://github.com/apache/incubator-nuttx/pull/6197

Reply:
BTW, we can move all crt0.c from arch near libs/libc/sched/task_startup.c and implement as this:
```
void _start(int argc, char *argv[])
{
  nxtask_startup(main, argc, argv);
}
```

Reply:
Maybe do the moving in a different patch